packages/core/src/tracing/span.ts

```diff
@@ -216,8 +218,8 @@ export class Span implements SpanInterface {
       const idStr = childSpan.transaction.spanId;
 
       const logMessage = `[Tracing] Starting '${opStr}' span on transaction '${nameStr}' (${idStr}).`;
-      childSpan.transaction.metadata.spanMetadata[childSpan.spanId] = { logMessage };
       logger.log(logMessage);
+      this._logMessage = logMessage;
     }
```

Comment:
I vote we kill this entire log-message-on-span thing. I am not sure it is worth the complexity. The actual logger log message should stay though probably.

Reply:
yeah, I generally agree, problem is we need the transaction name when finishing, where we don't really have access to this. IMHO this is OK for now and we can look to further simplify this when we do more work on spans (I don't want to spend too much time on this right now because... there are a million other things to do as well xD)

Reply:
Note that now the span simply keeps it's own log message so we can re-use it when it is ended, so it should already be much simpler than it was before 😅

Reply:
sounds good!